configure.ac

```diff
@@ -49,7 +49,7 @@ PKG_CHECK_MODULES(INITIAL_SETUP,
                   pango >= $PANGO_REQUIRED_VERSION
                   rest-0.7
                   json-glib-1.0
-		  libsecret-1
+                  libsecret-1
```

Comment:
I'd prefer not to carry this downstream patch. We can/should push a trivial fix upstream though.